static/templates/message_edit_form.hbs

```diff
@@ -16,7 +16,7 @@
                       list_placeholder=(t 'Filter streams')}}
                 </div>
                 <i class="fa fa-angle-right" aria-hidden="true" {{#unless is_stream_editable}}style="display:none"{{/unless}}></i>
-                <input type="text" placeholder="{{topic}}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" autocomplete="off" />
+                <input type="text" placeholder="{{topic}}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" autocomplete="off" {{#unless should_topic_be_visible}}style="display:none"{{/unless}} />
```

Comment:
Instead of using inline styles to hide the topic input, I think it'll be better to only include this input if the `should_topic_be_visible` conditional is true, using the handlebars if conditional syntax around the input

Reply:
Hmm, I too had a thought of this, I added a commit refactoring this method above of the one which fixes the issue.
Thanks!